Infrastructure/Factories/QualityAssuredSpreadsheetTuitionPartnerFactory.cs

```diff
@@ -76,12 +79,19 @@ public async Task<TuitionPartner> GetTuitionPartner(Stream stream, CancellationT
             Address = _spreadsheetExtractor.GetCellValue(GeneralInformationSheetName, "C", 9),
             Description = _spreadsheetExtractor.GetCellValue(GeneralInformationSheetName, "B", 13),
             Experience = _spreadsheetExtractor.GetCellValue(GeneralInformationSheetName, "C", 13),
-            LegalStatus = _spreadsheetExtractor.GetCellValue(GeneralInformationSheetName, "D", 13),
             HasSenProvision = _spreadsheetExtractor.GetCellValue(GeneralInformationSheetName, "F", 13).ParseBoolean(),
             AdditionalServiceOfferings = _spreadsheetExtractor.GetCellValue(GeneralInformationSheetName, "G", 13),
             IsVatCharged = _spreadsheetExtractor.GetCellValue(PricingSheetName, "K", 2).ParseBoolean(),
         };
 
+        //Get Organisation Type Id
+        var organisationTypeName = _spreadsheetExtractor.GetCellValue(GeneralInformationSheetName, "D", 13);
+        var organisationType = _organisationTypes!.Where(x => string.Equals(x.Name, organisationTypeName, StringComparison.InvariantCultureIgnoreCase)).FirstOrDefault();
```

Comment:
It can be simplified and replaced with a single call to FirstOrDefault.
`var organisationType = _organisationTypes!.FirstOrDefault(x => string.Equals(x.Name, organisationTypeName, StringComparison.InvariantCultureIgnoreCase));`

If we expect a unique value for the organisation type name, we can use `SingleOrDefault` instead of `FirstOrDefault` to enforce that.